apps/extension/src/rpc/rethrow-impl-errors.ts

```diff
@@ -12,10 +12,12 @@ const wrapUnaryImpl =
       const result = methodImplementation(req, ctx);
       if (result instanceof Promise)
         return result.catch((e: unknown) => {
+          if (process.env['NODE_ENV'] === 'development') console.debug(ctx.method.name, req, e);
```

Comment:
At first I thought the throw was within the if clause, can you add brackets to these?